internal/fips/nofips.go

```diff
@@ -0,0 +1,9 @@
+// Copyright 2021 - MinIO, Inc. All rights reserved.
+// Use of this source code is governed by the AGPLv3
+// license that can be found in the LICENSE file.
+
+// +build !fips
+
+package fips
+
+const enabled = 0 == 1
```

Comment:
Why use `0 == 1` instead of just `false`?

Reply:
This is kind of an excessive measurement but in Go `true` and `false` are valid identifiers.
The following is valid Go:
```go
package main

import (
	"fmt"
)

const true = false

func main() {
	fmt.Println(true) // Oh no...
}
```

Therefore, the "recommendation" to make really sure that a boolean constant is evaluated by compiler like it you expect is to use a boolean expression - like `0 == 0`. More of a compliance thing / theoretical discussion xD

Reply:
@aead is trying to be extra careful since its KES :-p

Reply:
Ah ok.